provider/meta_service/server/options/logging.go

```diff
@@ -0,0 +1,83 @@
+package options
```

Comment:
Suggest to move logging to common , can be to util directory

Reply:
@sushanthakumar logging options moved to ```utils```